src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
Lol yes. This is the place where @amazingant can plug in a very generic loader or something